lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_civil_partnership.govspeak.erb

```diff
@@ -42,6 +42,39 @@
+
+      ##What you need to do
+      <% if calculator.resident_outside_of_uk? %>
+
+        You don’t need a certificate of no impediment (CNI) to prove you’re allowed to marry if you’re a British national resident outside of the UK. However, you’ll need to apply for an exemption (‘Befreiung von der Beibringung des Ehefähigkeitszeugnisses’) from the Higher Regional Court (‘Oberlandesgericht’) in the region where you want to marry.
+
+        Contact the local registry office (‘Standesamt’) where you’re getting married - they’ll tell you what you need to do.
+
+        If the Standesamt does ask you for a CNI (‘Ehefähigkeitszeugnis’), refer them to the relevant part of the [German Civil Code](http://www.gesetze-im-internet.de/bgb/__1309.html) (‘Bürgerliches Gesetzbuch’).
+
+        You’ll need to complete a statutory declaration at the registry office as part of the application process.
+
+      <% elsif calculator.resident_of_uk? %>
+
+        You’ll be asked to provide a certificate of no impediment (CNI) to prove you’re allowed to enter into a civil partnership or equivalent in Germany.
+
+        You can normally get a CNI by giving a notice of marriage at your local register office or registrar in the UK. Find your local office or registrar in [England and Wales](https://www.gov.uk/register-offices), [Scotland](http://www.nrscotland.gov.uk/files/registration/reglist.pdf), [Northern Ireland](http://www.nidirect.gov.uk/index/information-and-services/government-citizens-and-rights/births-and-registration/district-registrars-in-northern-ireland.htm), [Isle of Man](http://www.gov.im/registries/general/civilregistry), [Jersey](http://www.gov.je/pages/contacts.aspx?contactId=71) or [Guernsey](http://www.guernseyroyalcourt.gg/article/1663/Contact-Us).
+
+        ^A CNI issued in Scotland is valid for 3 months. A CNI issued in England, Wales or Northern Ireland will not expire. Check with your local register office to find out how long a CNI is valid if you live in the Channel Islands or the Isle of Man. You should also check with the local authorities in Germany to find out how long a CNI is valid under local law.^
+
+        ###Translation
+
+        Check with the local authorities (standesamt) in Germany if you need to get your UK-issued CNI translated.
+
+        [Find a translator abroad](/government/collections/lists-of-translators-and-interpreters), or in the UK through the [Institute of Linguists](http://www.ciol.org.uk)
+
+        You should also check with the local authorities if you need to provide [legalised](/get-document-legalised) and translated copies of any other documents.
+
+        %The names on all documents you provide must appear exactly as they do on your passports - if not, the authorities may refuse to allow the marriage to go ahead. You may need to provide evidence if the name on your passport is different to your birth certificate (eg marriage certificate or deed poll).%
+
```

Comment:
It looks as though this "Translation" section is a duplicate of the content in [outcome_opposite_sex_marriage_in_germany](https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_germany.govspeak.erb#L26). It'd be awesome if you could remove that duplication somehow.